rotkehlchen/globaldb/schema.py

```diff
@@ -148,6 +160,49 @@
 );
 """
 
+DB_V3_CREATE_COMMON_ASSETS_DETAILS = """
```

Comment:
And this won't stay right? We willl only have 1 version of creation command per table.

Reply:
I was thinking in someone comming from an old version but I believe it doesn't make sense? We can go ahead and remove the versions in the names

Reply:
it doesn't make sense. We never keep old version creation schemas around. That's what DB upgrades are for.